myriad-scheduler/src/main/java/org/apache/myriad/configuration/MyriadConfiguration.java

```diff
@@ -197,47 +200,44 @@ public String getFrameworkRole() {
     return nmInstances;
   }
 
-  public Boolean isRebalancer() {
-    return rebalancer != null ? rebalancer : DEFAULT_REBALANCER;
+  public Boolean isRebalancerEnabled() {
+    return Optional.of(rebalancerEnabled).or(DEFAULT_REBALANCER_ENABLED);
```

Comment:
I think you want Optional.fromNullable no Optional.of, for the difference check out http://www.tutorialspoint.com/guava/guava_optional_class.htm.  In fact, here's the method:
```
  public static <T> Optional<T> of(T reference) {
    return new Present<T>(checkNotNull(reference));
  }
```
checkNotNull is a guava precondition which will though an exception.